gnomad_qc/v3/resources/meta.py

```diff
@@ -8,33 +8,28 @@
     VersionedTableResource,
 )
 
-from gnomad_qc.v3.resources.constants import CURRENT_RELEASE, CURRENT_VERSION, RELEASES
+from gnomad_qc.v3.resources.constants import CURRENT_VERSION, VERSIONS
 
 
 # Samples metadata
-def _meta_root_path(version: str = CURRENT_RELEASE) -> str:
+def _meta_root_path(version: str = CURRENT_VERSION) -> str:
     """
     Retrieve the path to the root metadata directory.
 
-    :param version: gnomAD release version
+    :param version: gnomAD version
     :return: String representation of the path to the root metadata directory
     """
     return f"gs://gnomad/metadata/genomes_v{version}"
 
 
-def meta_tsv_path(
-    version: str = CURRENT_RELEASE, meta_version: str = CURRENT_VERSION
-) -> str:
+def meta_tsv_path(version: str = CURRENT_VERSION) -> str:
     """
     Get the path to the finalized sample metadata information after sample QC.
 
-    :param version: gnomAD release version
-    :param meta_version: metadata version to return
+    :param version: gnomAD version
     :return: String path to the finalized metadata
     """
-    return (
-        f"{_meta_root_path(version)}/gnomad_v{version}_metadata_v{meta_version}.tsv.gz"
-    )
+    return f"{_meta_root_path(version)}/gnomad_v{version}_metadata.tsv.gz"
```

Comment:
I found that there's only gs://gnomad/metadata/genomes_v3/gnomad_v3_metadata_2019-09-27.tsv.gz, nothing inside gs://gnomad/metadata/genomes_v3.1/

Reply:
yeah, we never wrote it out to a tsv for v3.1, we don't really need it for anything unless someone asks us for it